v2/protobuf/config.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
```

Comment:
I haven't done much with grpc, what benefits do we have in comparison to using json for the message payload? Probably reduced payload size, but I don't think this has much impact here.